ray-operator/config/samples/pytorch-text-classifier/fine-tune-pytorch-text-classifier.py

```diff
@@ -0,0 +1,153 @@
+import ray
```

Comment:
I prefer to embed the Python script in the YAML file to maintain consistency between them. Ensuring the same commit for both the YAML and the Python file requires specifying the commit in the YAML's runtime environment. However, it's easy to forget updating the runtime environment whenever the Python script is modified.

Reply:
Should we just stick to the ConfigMap route then? It's easier but i don't want users thinking this is the recommended way to upload source for Ray jobs

Reply:
I'm inclined to keep the script separate because:

1) I like how to actual python script is versioned controled in a .py file intead of in embedded YAML 
2) The sample shows how to use working_dir in runtime environment.
3) This is more inline with how users will actually use RayJob (I imagine no one is adding their job code in a ConfigMap)

Reply:
>  i don't want users thinking this is the recommended way to upload source for Ray jobs

This makes sense to me. I will check with the Ray team to see if there is a repository where we can upload Ray example scripts.

Reply:
I checked with the Ray team, but we didn't reach a consensus on creating a new repository for the Python scripts. Hence, we can initially put them in the KubeRay repository. However, as the KubeRay repository is 186MB, downloading the script may take some time in the runtime environment.